src/plat/api/nvmem.rs

```diff
@@ -359,6 +367,6 @@ mod c_api {
     #[no_mangle]
     #[tracing::instrument(level = "trace", ret)]
     pub unsafe extern "C" fn _plat__GetNvSize() -> u32 {
-        super::NV_MEMORY_SIZE as u32
```

Comment:
Is NV_MEMORY_SIZE used anywhere else? Can it be removed? 
If it is used, where does the 32k value come from?